extras/vfd16lf01/LICENSE

```diff
@@ -0,0 +1,7 @@
+This is a driver for Samsung 16LF01 series VFD.
+
+Originate written by Daniele Napolitano, distributed under GPL.
```

Comment:
What In the case if I will get permission from author to use this code under other license ?